Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -259,11 +264,11 @@ private class TopSitesLayout: UICollectionViewLayout {
 
     private func setupForOrientation(orientation: UIInterfaceOrientation) {
         if orientation.isLandscape {
```

Comment:
I wonder if it's possible to achieve the same logic without using the orientation and only the size classes. Now that we have multi-window mode on iPad I'd like to stray away from relying on orientation if we can. Not sure if it's possible though.